jre/8/Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM openjdk:8u322-jre-slim
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends liblz4-1=1.8.3-1+deb10u1 libgcrypt20=1.8.4-5+deb10u1 libgnutls30=3.6.7-4+deb10u7 libhogweed4=3.4.1-1+deb10u1 libssl1.1=1.1.1d-0+deb10u8 openssl=1.1.1d-0+deb10u8 libgmp10=2:6.1.2+dfsg-4+deb10u1 && \
-    rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    liblz4-1=1.9.3-2 \
+    libgcrypt20=1.8.7-6 \
+    libgnutls30=3.7.1-5 \
+    libhogweed6=3.7.3-1 \
+    libssl1.1=1.1.1n-0+deb11u1 \
+    openssl=1.1.1n-0+deb11u1 \
+    libgmp10=2:6.2.1+dfsg-1+deb11u1 \
+    zlib1g=1:1.2.11.dfsg-2+deb11u1 \
```

Comment:
This one is new.
The latest scan (after fixed in this PR) found it
https://github.com/scalar-labs/docker/runs/5844891318?check_suite_focus=true